nodejs/eks/cluster.ts

```diff
@@ -122,7 +122,7 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         ingress: [],
         egress: [],
         tags: <aws.Tags>{
-            "Name": `${name}`,
+            "Name": `${name}-eksClusterSecurityGroup`,
```

Comment:
If we omit the postfix from the Name tag, both secgroups get tagged with the same name (clusterName), and it is the Name tag that is used for the display Name in the AWS Console; thus, requiring the user to look at the secgroup `Group Name` to distinguish the two, since their names are identical.